core/internal/helpers/sarama.go

```diff
@@ -281,8 +292,8 @@ func (c *BurrowSaramaClient) RefreshMetadata(topics ...string) error {
 // GetOffset queries the cluster to get the most recent available offset at the given time (in milliseconds) on the
 // topic/partition combination. Time should be OffsetOldest for the earliest available offset, OffsetNewest for the
 // offset of the message that will be produced next, or a time.
-func (c *BurrowSaramaClient) GetOffset(topic string, partitionID int32, time int64) (int64, error) {
-	return c.Client.GetOffset(topic, partitionID, time)
+func (c *BurrowSaramaClient) GetOffset(topic string, partitionID int32, timestamp int64) (int64, error) {
+	return c.Client.GetOffset(topic, partitionID, timestamp)
```

Comment:
nit: doesn't seem worth adding the name change here?

Reply:
You mean why s/time/timestamp? 

The linter complains because this change imports `time` which can overshadow.

Reply:
lols